packages/core/src/RenderPipeline/RenderQueue.ts

```diff
@@ -20,6 +26,50 @@ export class RenderQueue {
     return a.priority - b.priority || b.distanceForSort - a.distanceForSort;
   }
 
+  private static _getCustomStencilStates(
+    maskInteraction: SpriteMaskInteraction,
+    maskType: RenderQueueMaskType
+  ): Record<number, number | boolean> | null {
+    // When the renderer's mask interaction not none
+    if (maskInteraction !== SpriteMaskInteraction.None) {
```

Comment:
If `maskType` is not `No` and `maskInteraction` is not `No`
Do you think this will live up to expectations?

Reply:
> If `maskType` is not `No` and `maskInteraction` is not `No` Do you think this will live up to expectations?

yeap